lib/facter/ec2_tag_facts.rb

```diff
@@ -105,6 +124,22 @@ def debug_msg(txt)
 
           # append to the hash for structured fact later
           result[name] = tag['value']
+          if name =~ /env/
+		if tag['value'] == "prod"
+			subdomain = "#{region}.#{tag['value']}"
```

Comment:
I believe we agreed that all environments would have regions e.g. us-east-1.dev, us-east-1.qa, us-east-1.prod

Reply:
@shapi78 This is still a thing. All environments should be able to handle a region.